src/app/containers/App/App.jsx

```diff
@@ -15,14 +15,19 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
     errorCode,
     route: { pageType },
   } = getRouteProps(routes, location.pathname);
-
   const { pageData, status, error, timeOnServer } = initialData;
 
+  const pageLang = pageData.metadata.language;
+  let variantOverride;
+  if (service === 'ukrainian') {
```

Comment:
```suggestion
  if (service === 'ukrainian' && pageLang === 'ru') {
```

Reply:
Or alternatively:
```suggestion
  if(location.pathname === '/ukrainian/ukraine_in_russian') {
   variantOverride = 'ru';
  }
```

Reply:
As another alternative to doing this in the `App.jsx`, we could check the `pageType` and `service` in [this file](https://github.com/bbc/simorgh/blob/0ba61e622c138a99076c7ae8ae3ea4d95e65cc24/src/app/containers/PageHandlers/withContexts.jsx#L1) to apply the right variant.

Reply:
I agree with @DenisHdz that a specific check for this page path would be preferable.

Reply:
^ Removed my comment by mistake: 
```suggestion
  if (location.pathname === '/ukrainian/ukraine_in_russian') {
```

Reply:
so accepted pagesTypes that would be idx and sty?

Reply:
We opted for keeping this PR in scope of the ws-home team requirements for only the `'/ukrainian/ukraine_in_russian'` page